Healthcheck failed since FF deps removed in 1.0

### Current Behavior

return code not equal 0 in
```
$ node /healthcheck.js 
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '@flowforge/nr-auth'
Require stack:
- /data/settings.js
- /healthcheck.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.req
- /healthcheck.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/data/suire (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/data/settings.js:5:16)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/data/settings.js', '/healthcheck.js' ]
}
```

so containers are flagged "unhealthy"

### Expected Behavior

script in success (or disable healhcheck)

this is caused by removing FF nodes in commit ba745193022e0758f9d30ae91480eb55be068b31

I have worked arounded (not sustainable) by changing all "@flowforge/..." by absolute path in /data/settings.js
unfortunately I can't create a PR to fix it properly (perhaps multiples solutions to  fix this issue)

workaround example:
require('@flowforge/nr-auth') into require('/usr/src/flowforge-nr-launcher/node_modules/@flowforge/nr-auth') 

dependencies in /data/settings.js:

- nr-auth
- nr-storage
- nr-audit-logger

### Steps To Reproduce

with a FF-NR image >= 1.0 build

1. start a project
2. container is unhealthy

### Environment

- FlowForge version: 1.1
- Node.js version: v16.16.0
- npm version: 8.11.0
- Platform/OS: armv7
- Browser: not revelant
- FF/docker-compose >= 1.0

--- a/node-red-container/Dockerfile
+++ b/node-red-container/Dockerfile
@@ -6,6 +6,8 @@ ARG BUILD_TAG=latest
 RUN if [[ ! -z "$REGISTRY_TOKEN" ]]; then echo "//$REGISTRY/:_authToken=$REGISTRY_TOKEN" >> ~/.npmrc ; fi
 RUN if [[ ! -z "$REGISTRY" ]] ; then npm config set @flowforge:registry "https://$REGISTRY"; fi
 
+COPY healthcheck.js /healthcheck.js
+
 COPY package.json /data
 WORKDIR /data
 RUN npm install

--- a/node-red-container/healthcheck.js
+++ b/node-red-container/healthcheck.js
@@ -21,4 +21,4 @@ request.on('error', function(err) {
     process.exit(1);
 });
 
-request.end(); 
+request.end();

--- a/node-red-container/healthcheck.js
+++ b/node-red-container/healthcheck.js
@@ -1,0 +1,34 @@
+var http = require('http');
+var https = require('https');
+//var settings = require('/data/settings.js');
+var request;
+
+process.env["NODE_TLS_REJECT_UNAUTHORIZED"] = 0;
+
+var options = {
+    host : "localhost",
+    port : 1880,
+    timeout : 4000
+};
+
+if (settings.hasOwnProperty("https")) {
+    request = https.request(options, (res) => {
+        //console.log(`STATUS: ${res.statusCode}`);
+        if ((res.statusCode >= 200) && (res.statusCode < 500)) { process.exit(0); }
+        else { process.exit(1); }
+    });
+}
+else {
+    request = http.request(options, (res) => {
+        //console.log(`STATUS: ${res.statusCode}`);
+        if ((res.statusCode >= 200) && (res.statusCode < 500)) { process.exit(0); }
+        else { process.exit(1); }
+    });
+}
+
+request.on('error', function(err) {
+    //console.log('ERROR',err);
+    process.exit(1);
+});
+
+request.end(); 

--- a/node-red-container/healthcheck.js
+++ b/node-red-container/healthcheck.js
@@ -1,30 +1,20 @@
 var http = require('http');
 var https = require('https');
-//var settings = require('/data/settings.js');
 var request;
 
 process.env["NODE_TLS_REJECT_UNAUTHORIZED"] = 0;
 
 var options = {
     host : "localhost",
-    port : 1880,
+    port : 2880,
     timeout : 4000
 };
 
-if (settings.hasOwnProperty("https")) {
-    request = https.request(options, (res) => {
-        //console.log(`STATUS: ${res.statusCode}`);
-        if ((res.statusCode >= 200) && (res.statusCode < 500)) { process.exit(0); }
-        else { process.exit(1); }
-    });
-}
-else {
-    request = http.request(options, (res) => {
-        //console.log(`STATUS: ${res.statusCode}`);
-        if ((res.statusCode >= 200) && (res.statusCode < 500)) { process.exit(0); }
-        else { process.exit(1); }
-    });
-}
+request = https.request(options, (res) => {
+    //console.log(`STATUS: ${res.statusCode}`);
+    if ((res.statusCode >= 200) && (res.statusCode < 500)) { process.exit(0); }
+    else { process.exit(1); }
+});
 
 request.on('error', function(err) {
     //console.log('ERROR',err);

